include/NovelRT/Atom.h

```diff
@@ -55,54 +55,24 @@ namespace NovelRT
             return _value >= other._value;
         }
 
-        operator uintptr_t() noexcept
+        operator uintptr_t() const noexcept
```

Comment:
This not being `const` meant that C/C++ thought it was mutating the value and so it couldn't be used with a `const Atom`.